src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+
+\begin{figure}
+	\import{code-examples/generated/}{gp_maps}
+
+	\caption{
+        Using \code{gammapy.maps} to create a WCS, a HEALPix and a region
+		based data structures. The initialisation parameters include
+        consistently the positions of the center of the map, the pixel
+        size, the extend of the map as well as the energy axis definition.
+        The energy minimum and maximum values for the creation of the
+        \code{MapAxis} object can be defined as strings also specifying the
+        unit. Region definitions can be passed as strings following
+        the DS9 region specifications \url{http://ds9.si.edu/doc/ref/region.html}.
+        }
+    \label{ig*:minted:gp_maps}
+\end{figure}
+
+% itemize because it helps me write, could also just be paragraphs...
+\subsubsection{WCS Maps}
+The FITS WCS pixelization supports a different number of projections to
+represent celestial spherical coordinates in a regular rectangular grid.
+\gammapy provides full support to data structures using this pixelization
+scheme. For details see ~\cite{Calabretta2002}. This pixelisation
+is typically used for smaller regions of interests, such as pointed
+observations.
+
+
+\subsubsection{HEALPix Maps}
+This pixelization scheme ~\citep{Calabretta2002} provides a
+subdivision of a sphere in which each pixel covers the same surface area as
+every other pixel. As a consequence, however, pixel shapes are no longer
+rectangular, or regular.
+This pixelisation is typically used for all-sky data, such as data
+from the \hawc or \fermi observatory. \gammapy natively supports
+the multiscale definiton of the HEALPix pixelisation and thus
+allows for easy up and downsampling of the data. In addition to
+the all-sky map, \gammapy also supports a local HEALPix
+pixelisation where the size of the map is constrained to a given
+radius.
+For local nighbourhood operations, such as convolution \gammapy relies
+on projecting the HEALPix data to a local tangential WCS grid.
+
+\subsubsection{Region Maps}
+In this case, instead of a fine spatial grid
+dividing a rectangular sky region, the spatial dimension is reduced to a single
+bin with an arbitrary shape, describing a region in the sky with that same
+shape. Typically they are is used together with a non-spatial dimension, for
+example an energy axis, to represent how a quantity varies in that dimension
+inside the corresponding region. The region is represented on the local
+tangential projection see \cite{Astropy regions}.
+
+Additionally, the \code{MapAxis} class provides a uniform API for axes representing
+bins on any physical quantity, such as energy or angular offset. The special
+case of time is covered by the dedicated \code{TimeMapAxis}, which allows time bins to
+be non-contiguous, as it is often the case with observation time-stamps. The
+generic class \code{LabelMapAxis} allows the creation of axes for non-numeric
+entries.
+
+\subsection{gammapy.datasets}
```

Comment:
For a non-expert, this paragraph is hard to read... and for Atreyee hard to write, because one speaks about statistics, Cstat, Wstat, likelihood, etc without defining it. In the reading flow, it is not great...
I am wondering whether one should/could move the sub-section `gammapy.stats` just before this sub-section? Then it could be simplied a bit